plugins/conftest.py

```diff
@@ -0,0 +1,29 @@
+from os import path
+
+import pytest
+import six
+
+
+@pytest.fixture()
+def module_initializer(request):
```

Comment:
Amazing. Great work!

Reply:
Thanks! :) It would be nice if we didn't have to do this though, but I think it's the only way without re-doing the method for loading plugins.